data/scripts.js

```diff
@@ -1715,29 +1715,31 @@ exports.BattleScripts = {
 			item = 'Rocky Helmet';
 		} else if (counter.Physical + counter.Special >= 4 && template.baseStats.def + template.baseStats.spd > 189) {
 			item = 'Assault Vest';
-		} else if (counter.Physical + counter.Special >= 4) {
+		} else if (counter.Physical + counter.Special >= 4 && ability === 'Multiscale') {
```

Comment:
I'm pretty sure Dragonite/Lugia will never get Life Orb from here. Lugia doesn't have four attacks, and Dragonite doesn't have a special move so it will get Choice from a check above.